docs/genai/howto/install.md

```diff
@@ -23,61 +27,120 @@ The CUDA toolkit can be downloaded from the [CUDA Toolkit Archive](https://devel
+### DirectML
 
 ```bash
-pip install numpy
+pip install onnxruntime-directml
 pip install onnxruntime-genai-directml --pre
 ```
 
 ### CUDA
 
-Append `-cuda` for the library that is optimized for CUDA environments
-
 #### CUDA 11
 
 ```bash
-pip install numpy
-pip install onnxruntime-genai-cuda --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-genai/pypi/simple/
+# Install ORT nightly CUDA 11, change this to released version when it is released
+pip install ort-nightly-gpu --index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-cuda-11-nightly/pypi/simple
+
+
+# Install onnxruntime-genai built for CUDA 11
+pip install onnxruntime-genai-cuda-11 --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-genai/pypi/simple/
 ```
 
 #### CUDA 12
 
 ```bash
-pip install numpy
-pip install onnxruntime-genai-cuda --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/
+# Install ORT nightly CUDA 12, change this to released version when it is released
+pip install onnxruntime-gpu --extra-index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/
+
+# Install onnxruntime-genai build for CUDA 12
+pip install onnxruntime-genai-cuda --pre --index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-genai/pypi/simple/
 ```
 
 ## Nuget packages
 
-Note: only one of these packages should be installed in your application.
+Note: only one of these set of packages (CPU, DirectML, CUDA) should be installed in your application.
+
+### CPU
 
 ```bash
+dotnet add package Microsoft.ML.OnnxRuntime
 dotnet add package Microsoft.ML.OnnxRuntimeGenAI --prerelease
 ```
 
-For the package that has been optimized for CUDA:
+### CUDA 
+
+#### CUDA 11
+
+Add the following lines to a `nuget.config` file in the same folder as your `.csproj` file.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <packageSources>
+        <clear/>
+        <add key="onnxruntime-cuda-11"
+             value="https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-11/nuget/v3/index.json"/>
+    </packageSources>
+</configuration>
+```
+
+```bash
+dotnet add package Microsoft.ML.OnnxRuntime.Gpu
+dotnet add package Microsoft.ML.OnnxRuntimeGenAI.Cuda --prerelease
+```
+
+#### CUDA 12
+
+Add the following lines to a `nuget.config` file in the same folder as your `.csproj` file.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <packageSources>
+        <clear/>
+        <add key="onnxruntime-cuda-12"
+             value="https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/nuget/v3/index.json"/>
+    </packageSources>
+</configuration>
+```
+
 
 ```bash
+dotnet add package Microsoft.ML.OnnxRuntime.Gpu
```

Comment:
Manually adding `Microsoft.ML.OnnxRuntime.Gpu` and `Microsoft.ML.OnnxRuntime.DirectML` is technically unnecessary, because GenAI NuGet has dependency of the corresponding ORT packages.

Reply:
I think? we still need to install it explicitly to get nightly but let me know if there is an easier way to do it

Reply:
We can auth our way to the nightly feed. but I'm not sure if users outside MSFT *can* access the nightly feed. The pip feed is OK, but with recent security overhaul, the NuGet feed seems inaccessible outside MSFT.

Secondly, I'm not sure if nightly version will meet the requirement in GenAI nuspec. For example, if we specify `ORT=1.19.0`, will it accept `ORT=1.20.0-dev`? This is the first time we explicitly depend on ORT in the GenAI nuspec and I'm no NuGet expert. My initial thought was to depend only on the official release of ORT. If nightly is a must-have, we need to investigate more.

Reply:
You were right - the dependencies have been added to the GenAI package. I have removed the separate install. And tested with the ADO source.